src/Components/benchmarkapps/Wasm.Performance/Driver/SeleniumServer.cs

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Http;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Wasm.Performance.Driver
+{
+    class SeleniumServer : IDisposable
+    {
+        private SeleniumServer(Process process)
+        {
+            SeleniumProcess = process;
+        }
+
+        private Process SeleniumProcess { get; }
+
+        public static SeleniumServer Start(int port = 4444)
+        {
+            var outputLock = new object();
+
+            var psi = new ProcessStartInfo
+            {
+                FileName = "/opt/bin/start-selenium-standalone.sh",
```

Comment:
???

Is this only intended to run within a particular Docker image? If so perhaps this could be clarified via the naming of the project or something.

Reply:
I don't really understand why we need a Linux-specific version of "start selenium" - don't we already have a cross-plat version of this logic in the E2E test infrastructure?